package.json

```diff
@@ -7,23 +7,29 @@
     "prestart": "run-p clean node-version",
     "start": "run-p build:dev serve:dev",
     "postserve:dev": "docker-compose down",
-    "clean": "rm -rf theme/",
-    "copy": "mkdir -p ./theme && cp -r ./src/php/* ./theme",
+    "clean": "rm -rf theme/ plugins/example-blocks/",
+    "copy": "mkdir -p ./theme && cp -r ./src/php/* ./theme && mkdir -p ./plugins && cp -r ./src/plugins/* ./plugins",
```

Comment:
I think it probably makes sense to split this into two scripts: `copy:php` and `copy:plugins`. We only need the PHP files to be copied during development, which is triggered by our `watch:php` script. The plugins are handled by the volume mapping in development, so we only need to copy them manually for `build:prod`.

Reply:
Ok, I separated the scripts into `copy:php` and `copy:plugins`, and `copy` will run both, which will only happen with `build:prod`, otherwise for `build:dev` and `watch:php` it will only run `copy:php`. Does that sound correct? Did I miss anything?

Reply:
This all looks correct to me, I think we're good 👍